### DISCUSSION TITLE
Which .deb file should I to install Motion Project in a Ubuntu 18.04 32 bits of an x86 computer?

### DISCUSSION THREAD

**Discussion by thosecars82** [Motion-Project/motion#1770] [Q&A]
Which .deb file should I to install Motion Project in a Ubuntu 18.04 32 bits of an x86 computer?
Thanks

**Comment by tosiara** [comment]
You should use: bionic_motion_4.6.0-1_i386.deb 
https://github.com/Motion-Project/motion/releases/download/release-4.6.0/bionic_motion_4.6.0-1_i386.deb

**Reply by thosecars82** [reply]
Thanks, browsing http://localhost:8080/ opened the web interface.
However, it does not find the Tp-Link Tapo C200, which is in the same LAN.

**Reply by tosiara** [reply]
Motion will not detect anything, you must put the camera URL into config. Please check the documentation
